src/layout/Error.tsx

```diff
@@ -0,0 +1,198 @@
+import React from 'react';
+import type { HtmlHTMLAttributes } from 'react';
+import { Title, useTranslate } from 'react-admin';
+import type { ErrorProps } from 'react-admin';
+import {
+  Accordion,
+  AccordionDetails,
+  AccordionSummary,
+  Box,
+  Button,
+  Typography,
+} from '@mui/material';
+import { styled } from '@mui/material/styles';
+import ExpandMoreIcon from '@mui/icons-material/ExpandMore';
+import HistoryIcon from '@mui/icons-material/History';
+import RefreshIcon from '@mui/icons-material/Refresh';
+import type { FallbackProps } from 'react-error-boundary';
+import LogoError from './LogoError';
+
+const PREFIX = 'RaError';
+
+export const ErrorClasses = {
+  container: `${PREFIX}-container`,
+  title: `${PREFIX}-title`,
+  logo: `${PREFIX}-logo`,
+  panel: `${PREFIX}-panel`,
+  panelSummary: `${PREFIX}-panelSummary`,
+  panelDetails: `${PREFIX}-panelDetails`,
+  toolbar: `${PREFIX}-toolbar`,
+  advice: `${PREFIX}-advice`,
+};
+
+interface InternalErrorProps
+  extends Omit<HtmlHTMLAttributes<HTMLDivElement>, 'title'>,
+    Partial<Pick<FallbackProps, 'resetErrorBoundary'>>,
+    ErrorProps {
+  className?: string;
+}
+
+// eslint-disable-next-line tree-shaking/no-side-effects-in-initialization
+const Root = styled('div', {
```

Comment:
If you don't intend to make this component overiddable , don't bother with `styled`, and use `sx` diurectly in the JSX code.